src/api/app/assets/stylesheets/webui/application/request.scss

```diff
@@ -8,4 +8,14 @@
 }
 .request-critical {
   color: #d80000;
-}
\ No newline at end of file
+}
+
+.review_descision_display {
```

Comment:
Top late but why not just this:

``` css
[id^='review_accept_button']{
  margin-right: 1em;
}
```

I don't like to set the `margin-right` for all inputs with submit type inside `review_decision_display` and then remove it for the `Disregards` button using that it is the last one. In my opinion it is better to directly set the margin to the `Approve` button.

@DavidKang @bgeuken what do you think?